practice-2/count_word_book.py

```diff
@@ -0,0 +1,82 @@
+"""
+This module contains a code for exercises 13-2 related to:
+Think Python, 2nd Edition
+Chapter 13: Case Study: Data Structure Selection
+
+Modify the program to count the total number of words in the book, and the
+number of times each word is used. Print the number of different words used in the book.
+
+Download Ebook.txt file
```

Comment:
I see you put the file names in the texts folder out of sync, `Ebook.txt` capitalizes the first letter, but the rest of the files don't.